definitions/variable/technology/power-plant.yaml

```diff
@@ -125,86 +127,63 @@
-    description: Load factors for onshore wind power plants
+    description: Load factors for {Variable Renewable Power Type} power plants
     unit: '%'
     skip-region-aggregation: true
-- Maximum Energy Charge|Electricity|Energy Storage System:
-    description: This is the upper bound that energy storage system could charge energy
-      at time "t".
-    unit: GWh
-- Maximum Charge|Electricity|Energy Storage System:
-    description: Maximum charge when storing energy the energy storage system unit.
-    unit: GW
-- Roundtrip Efficiency|Electricity|Energy Storage System|{Energy Storage System Type}:
+- Roundtrip Efficiency|Electricity|{Electricity Storage Type}:
     description: Efficiency when charging energy from the grid to the system.
-    unit: 
-- Maximum Discharge|Electricity|Energy Storage System:
-    description: Maximum discharge when consuming energy from the energy storage system
-      unit.
-    unit: GW
-- Maximum Storage|Electricity|Energy Storage System:
-    description: Maximum energy that can be stored by the energy storage systems.
-    unit: GWh
-- Maximum Storage|Electricity|Energy Storage System|{Energy Storage System Type}:
-    description: Maximum energy that can be stored by the Lithium Iron Batteries systems.
-    unit: GWh
-- Minimum Storage|Electricity|Energy Storage System:
-    description: Minimum energy that can be stored by the energy storage system unit.
-    unit: GWh
-- Spillage|Electricity|Energy Storage System:
-    description: Energy spillage that could not be stored by an energy storage system
-      (ESS). Several energy models include spillage variables related to renewable
+    unit:
+- Spillage|Electricity|{Electricity Storage Type}:
+    description: Energy spillage that could not be stored by a {Electricity Storage Type} unit. Several energy models include spillage variables related to renewable
       energy production; however, another way is to include a spillage variable to
-      the ESS unit since it is the last sink of the power network.
-    unit: GWh
-- Seasonality|Electricity|Energy Storage System|{Energy Storage System Type}:
-    description: seasonality (daily, weekly, monthly, etc.) of {Energy Storage System Type}
-    unit: [daily,weekly,monthly]
-- Seasonality|Electricity|Hydro|Reservoir:
-    description: seasonality (daily, weekly, monthly, etc.) of Reservoir
-    unit: [daily,weekly,monthly]
-- Seasonality|Electricity|Hydro|Pumped Storage:
-    description: seasonality (daily, weekly, monthly, etc.) of Pumped Storage
-    unit: [daily,weekly,monthly]
+      the {Electricity Storage Type} unit since it is the last sink of the power network
+    unit:
+    - MW
+    - GW
+- Storage Type|Electricity|{Electricity Storage Type}:
+    description: Storage type (daily, weekly, monthly, etc.) of a {Seasonality} unit
+    unit:
+    - Daily
+    - Weekly
+    - Monthly
+- Stored Energy Inventory|Electricity|{Electricity Storage Type}:
```

Comment:
I suggest to remove that variable as it is exactly the same thing as the variable 'Storage' in electricity-operation (plus also it should be in electricity-operation as it is not a fixed characteristic of a unit bt a result of operation)